GsaGHTests/2_GooWrappers/GsaElement2dGooTests.cs

```diff
@@ -1,32 +1,108 @@
 ﻿using Grasshopper.Kernel.Types;
 
+using GsaAPI;
+
 using GsaGH.Components;
 using GsaGH.Parameters;
 
 using GsaGHTests.Components.Geometry;
 using GsaGHTests.Helpers;
+using GsaGHTests.Parameters;
+
+using Rhino.Geometry;
+using Rhino.Geometry.Morphs;
 
 using Xunit;
 
 namespace GsaGHTests.GooWrappers {
   [Collection("GrasshopperFixture collection")]
   public class GsaElement2dGooTests {
-    [Fact]
-    public void GsaElement2dGooDrawViewportMeshesAndWiresTest() {
-      var comp = (Section3dPreviewComponent)CreateElement2dTests.ComponentMother();
+    [Theory]
+    [InlineData(true)]
+    [InlineData(false)]
+    public void GsaElement2dGooDrawViewportMeshesAndWiresTest(bool isLoadPanel) {
+      var comp = (Section3dPreviewComponent)CreateElement2dTests.ComponentMother(isLoadPanel, isLoadPanel);
```

Comment:
Specify the type of the `ComponentMother` so we avoid casting.

Also move this common setup to the constructor